app/services/dialog_local_service.rb

```diff
@@ -88,6 +99,8 @@ def determine_api_endpoints(obj, display_options = {})
 
   def determine_target_type(obj)
     case obj.class.name.demodulize
+    when /^Ebs/
+      "ems_storage"
```

Comment:
Should there be a spec for this case?

Reply:
This line is really troubling me.

Matching a class name by prefix....

Someone will have a hell of a job to refactor that one day.